static/js/book_borrowing_dashboard.js

```diff
@@ -0,0 +1,24 @@
+    function reserved_books() {
+        document.getElementById("booked_table").style.display = "none";
+        document.getElementById("reserved_table").style.display = "block";
+        document.getElementById("reserv_button").style.textDecoration = "underline";
+        document.getElementById("booked_button").style.textDecoration = "none";
+    }
+    function booked_books() {
+        document.getElementById("reserved_table").style.display = "none";
+        document.getElementById("booked_table").style.display = "block";
+        document.getElementById("reserv_button").style.textDecoration = "none";
+        document.getElementById("booked_button").style.textDecoration = "underline";
+    }
```

Comment:
We are using Jquery in this project, so use: $("#myId").css('display', 'none') etc